src/react/components/common/keyboardManager/keyboardManager.tsx

```diff
@@ -24,7 +24,7 @@ export class KeyboardManager extends React.Component<any, IKeyboardContext> {
     };
 
     private nonSupportedKeys = new Set(["Ctrl", " Control", "Alt"]);
-    private inputElementTypes = new Set(["input", "textarea"]);
+    private inputElementTypes = new Set(["input", "select", "textarea"]);
```

Comment:
Switching options in the select element can be done via keyboard as well, so it should be in the "ignoring" list.

Reply:
Thanks for added... i've added a couple more tests.